shippo/api_requestor.py

```diff
@@ -162,7 +162,11 @@ def interpret_response(self, rbody, rcode):
         try:
             if hasattr(rbody, 'decode'):
```

Comment:
instead of this extra if to manipulate the rbody, can you check if rbody is not '' in the first if where it checks the hasattr?